.github/workflows/gitlab-runners-ubi8-git-publish.yaml

```diff
@@ -0,0 +1,37 @@
+name: ubi8-git-publish
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - gitlab-runners/ubi8-git/version.json
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+    # Only build on redhat-cop repo, do not attempt to build on forks
+    if: github.repository == 'redhat-cop/containers-quickstarts'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get image tags
+        id: image_tags
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          VERSION=$(jq -r '.version' ${context}/version.json)
+          if [ "${VERSION}" == "latest"]; then
+            echo latest
+          else
+            echo "latest,${VERSION}"
+          fi
+      - name: Build and publish image to Quay
+        uses: elgohr/Publish-Docker-Github-Action@master
```

Comment:
looks like docker provides an offical action to push: https://www.docker.com/blog/first-docker-github-action-is-here/ - any reason not to use that vs this community one?

Reply:
No reason, changed to `docker/build-push-action`